Bugzilla/Install/Filesystem.pm

```diff
@@ -81,12 +82,16 @@ EOT
 
 use constant HT_ASSETS_DIR => <<'EOT';
 # Allow access to .css and js files
-<FilesMatch \.(css|js)$>
-  Allow from all
+<FilesMatch state\.json$>
+    Deny from all
 </FilesMatch>
 
+FileETag None
```

Comment:
I assume an Etag is set elsewhere in the code (even just the string "1" would be fine), as otherwise this may cause undesirable caching behavior from clients.